README.md

```diff
@@ -189,11 +196,34 @@ We can run `openssl crl -in ./cert/pki/crl.pem -noout -text` to validate that th
 
 To generate a new CRL:
 
-* Decrypt ca.key `sops -d ./cert/pki/private/ca.key.enc > ./cert/pki/private/ca.key`
+* Decrypt the required files as described in [HowTo Decrypt the Certificate Authority Key](#howto-decrypt-the-certificate-authority-key)
 * Generate a new crl.pem - `cd cert ; ./easyrsa gen-crl ; cd ..`
 * Publish the new crl.pem - `git add ./cert/pki/crl.pem && git commit ./cert/pki/crl.pem -s -m 'Renew revocation list certificate'`
 * Delete local ca.key - `rm ./cert/pki/private/ca.key`
 
+### How to Renew Server-side Certificate?
+
+* Build EASYVPN binary by running one of the following commands depending on your
+  * `make init_osx`
+  * `make init_linux`
+  * `make init_windows` and copy `./utils/easyvpn/easyvpn.exe` at the root of this repository
+* Decrypt the required files as described in [HowTo Decrypt the Certificate Authority Key](#howto-decrypt-the-certificate-authority-key)
+* Revoke actual certificate (even if it is already expired): `./easyvpn revoke vpn.jenkins.io`
+* Generate a new certificate + key, with the server DNS as argument: `./easyvpn request vpn.jenkins.io`
+
+  > The generated key is in `./cert/pki/private/vpn.jenkins.io.key` **must** remain **secret**!
+
+* Sign the request as a "server" request:
+
+  ```shell
+  cd ./certs # Mandatory to run the signing command from this repository
+  ./easyrsa --batch sign-req server vpn.jenkins.io
+  ```
+
+* Ensure that you git-added, git-commited and pushed the changes, without ANY secrets (should be git-ignored)
+
+* Update the secrets in the encrypted hieradata for OpenVPN in <https://github.com/jenkins-infra/jenkins-infra>
```

Comment:
```suggestion
* Update the secrets in the encrypted hieradata for OpenVPN in link:https://github.com/jenkins-infra/jenkins-infra/blob/0d6117dbd233a4c8cf5f29cc58821b9a71086f64/hieradata/clients/vpn.jenkins.io.yaml#L14-L21[github.com/jenkins-infra/jenkins-infra]
```

Reply:
I prefer to not detail too much: the permalink in this case is a "snapshot" at a given moment in time, while I expect the reader to fill the gaps when running this operation.